### DISCUSSION TITLE
Resume training while changing the total epochs in config file

### DISCUSSION THREAD

**Discussion by gaopinghai** [open-mmlab/mmengine#799] [Q&A]
Hi, is it possible to resume training while increasing the total epochs in config file? That's to say the total epochs is 100 and the training process is done. Then I want to increase the total epochs to 200, and I want to resume from the 100th epoch. How could I do this?

**Comment by HAOCHENYE** [comment]
"I haven't tried it yet, but I think you maybe need to do the following things:

1. Change the `max_epochs` in `train_cfg`
2. Manually change the `begin` and `end` in `param_scheduler`
3. Set `resume=True`"
